packages/FlexGrid/Col/Col.jsx

```diff
@@ -5,46 +5,114 @@ import { Subscriber } from 'react-broadcast'
 import { Col as ReactFlexboxGridCol } from 'react-flexbox-grid'
 
 import safeRest from '../../../shared/utils/safeRest'
+import { deprecate } from '../../../shared/utils/warn'
 import styles from './Col.modules.scss'
 
-const removeProps = ({
-  xs,
-  sm,
-  md,
-  lg,
-  xl,
+const removeProps = ({ ...rest }) => safeRest(rest)
+
+const Col = ({
+  span,
+  offset,
   xsOffset,
   smOffset,
   mdOffset,
   lgOffset,
   xlOffset,
+  xs,
+  sm,
+  md,
+  lg,
+  xl,
+  children,
   ...rest
-}) => safeRest(rest)
-
-const Col = ({ span, offset, children, ...rest }) => (
-  <Subscriber channel="flex-grid">
-    {gutter => (
-      <ReactFlexboxGridCol
-        {...removeProps(rest)}
-        xs={span || true}
-        xsOffset={offset}
-        className={gutter ? styles.padding : styles.gutterless}
-      >
-        {children}
-      </ReactFlexboxGridCol>
-    )}
-  </Subscriber>
-)
+}) => {
+  const flexProps = {}
+
+  if (sm) flexProps.sm = sm
+  if (md) flexProps.md = md
+  if (lg) flexProps.lg = lg
+  if (xl) flexProps.xl = xl
```

Comment:
I wonder if we making this harder on ourselves here...

Seems like the `PropTypes` definitions below will take care of showing the right interface for the docs, so thats fine.

Would an better solution be to just remove all the responsive and offset props from the component declaration, so that they all get captured by `rest`, and then just do the normal `...safeRest(rest)` to pass everything directly to the underlying react-flexbox-grid component? Then all of this logic with `flexProps` can go away. I think you would also remove the `defaultProps` stuff for all the react-flexbox-grid props too so that exactly what the user specifies will just get forwarded on.

If the linter complains, I'd say in this case its ok to disable the linter check for those props because we have a good reason.

Reply:
I tried this, and weirdly, React will spit out this error: `Error: A cross-origin error was thrown. React doesn't have access to the actual error object in development. See https://fb.me/react-crossorigin-error for more information.` Which is both strange and not very helpful.

Reply:
I think we can fix that, probably just some syntax error or something